common/appscale/common/monit_app_configuration.py

```diff
@@ -31,6 +32,8 @@ def create_config_file(watch, start_cmd, pidfile, port=None, env_vars=None,
       megabytes that the process should use.
     syslog_server: The IP address of the remote syslog server to use.
     check_port: A boolean specifying that monit should check host and port.
+    kill_exceeded_memory: A boolean indicating that a process should be killed
+      (instead of terminated) if its memory exceeds the limit.
```

Comment:
I think the comment should be something like: ... instead of terminated). This is mainly used when the process exceed its memory limits. Reason is the memory limit is not checked in this script, but somewhere else.